src/blob/core.rs

```diff
@@ -244,6 +245,25 @@ impl Blob {
         }))
     }
 
+    #[inline]
```

Comment:
I think it's redundant here, because won't add any performance, but will increase compilation time